Common/TaskCreator.cs

```diff
@@ -5,15 +5,15 @@ namespace Common
 {
     public static class TaskCreator
     {
-        public static ITask CreateTask(string assemblyName, string className)
+        public static ISubtask CreateTask(string assemblyName, string className)
```

Comment:
Let's change the name of the method to `CreateSubtask` so as not to use a possibly misleading name